src/ResourceManagement/DocumentDB/Microsoft.Azure.Management.DocumentDB.Fluent/DatabaseAccountImpl.cs

```diff
@@ -89,7 +90,15 @@ private Models.DatabaseAccountCreateUpdateParametersInner CreateUpdateParameters
 
         private bool IsAFinalProvisioningState(string state)
```

Comment:
the more standard sounding naming I think would be `IsProvisioningStateFinal(...)`